dune

```
@@ -0,0 +1,17 @@
+(library
+ (public_name zip)
+ (synopsis "OCaml ZIP interface")
+ (modules gzip zip zlib)
+ (libraries stdlib-shims)
```

```suggestion
 (modules gzip zip zlib)
 (wrapped false)
 (libraries stdlib-shims)
```

I believe `(wrapped false)` is necessary to avoid changing the API. (Thanks for this patch :slightly_smiling_face:)

---

Yes, this is what I meanwhile use in my `ocaml-camlzip.patch` file in the [openSUSE ocaml-camlzip](https://build.opensuse.org/package/show/devel:languages:ocaml/ocaml-camlzip) rpm package

---

(Just discovered that there's a `zlib` package already in Opam, which this conflicts with.)